dev-libs/libxml2/libxml2-2.12.3.recipe

```diff
@@ -8,14 +8,14 @@ markup language.
 Though the library is written in C a variety of language bindings make it \
 available in other environments."
 HOMEPAGE="http://www.xmlsoft.org/"
-COPYRIGHT="1998-2013 Daniel Veillard.  All Rights Reserved."
+COPYRIGHT="1998-2012 Daniel Veillard.  All Rights Reserved."
 LICENSE="MIT"
-REVISION="3"
-SOURCE_URI="https://download.gnome.org/sources/libxml2/2.10/libxml2-$portVersion.tar.xz"
-CHECKSUM_SHA256="ed0c91c5845008f1936739e4eee2035531c1c94742c6541f44ee66d885948d45"
+REVISION="1"
+SOURCE_URI="https://download.gnome.org/sources/libxml2/2.12/libxml2-$portVersion.tar.xz"
+CHECKSUM_SHA256="8c8f1092340a89ff32bc44ad5c9693aff9bc8a7a3e161bb239666e5d15ac9aaa"
 PATCHES="libxml2-$portVersion.patchset"
 
-ARCHITECTURES="all"
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
Well, that won't work. We need a GCC2 version.

What happens when trying to build with it? Did they switch off C89?

Reply:
@kenmays did you check a build with gcc2?

Reply:
Didn't check build on GCC2.